src/components/Table.res

```diff
@@ -267,7 +267,7 @@ module SortAction = {
     ~isLastCol=false,
     ~filterRow: option<filterRow>,
   ) => {
-    if item.showSort || filterRow->Option.isSome {
+    let _ = if item.showSort || filterRow->Option.isSome {
```

Comment:
why let _ = ?

Reply:
let's say if it's a very small table let's say profile id table over that place we can have a local sort instead of going to BE for that. Can you please check this case. We shouldn't be calling API for sort for every table.